test/conformance/cgroup_test.go

```diff
@@ -0,0 +1,100 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"testing"
+
+	"github.com/knative/pkg/test/logging"
+	"github.com/knative/serving/test"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestMustCgroupConfigured verifies that the Linux cgroups are configured based on the specified
+// resource limits and requests as delared by "MUST" in the runtime-contract.
+func TestMustCgroupConfigured(t *testing.T) {
```

Comment:
```suggestion
func TestMustHaveCgroupConfigured(t *testing.T) {
```
perhaps? Reads better, I think. Though I get, that `must` is from the spec.